src/transformers/generation/__init__.py

```diff
@@ -110,8 +110,6 @@
         "BeamSearchDecoderOnlyOutput",
         "BeamSampleEncoderDecoderOutput",
         "BeamSampleDecoderOnlyOutput",
-        "ContrastiveSearchEncoderDecoderOutput",
```

Comment:
let's leave the output class definitions in the main import (see the following comment in `generate/utils.py`: "# TODO (joao): remove the equivalent classes and typing shortcuts below in v5" -- there is a surprising number of libraries that expect to import these 👀 )